static/app/views/performance/newTraceDetails/trace.tsx

```diff
@@ -401,9 +401,7 @@ export function Trace({
       >
         {trace.indicators.length > 0
           ? trace.indicators.map((indicator, i) => {
-              const score = indicator.score
-                ? Math.round(indicator.score * 100)
-                : undefined;
+              const score = indicator.score ? indicator.score : undefined;
               const status = score ? STATUS_TEXT[scoreToStatus(score)] : 'none';
```

Comment:
This will also correct an issue if score is 0 and would have previously been reported as none
```suggestion
              const status = score === undefined ? 'none' :  STATUS_TEXT[scoreToStatus(score)];
```